examples/panel/stock-market-chatbot/README.md

```diff
@@ -0,0 +1,33 @@
+## LLM-powered Stock Market app with Panel
+
+This application downloads stock market data from Yahoo Finance, populates a duckdb instance, generates a time series plot of the selected stocks. It also allows the user to ask a natural language question about the plot and get a response using the LLM model.
+
+The app is built using Python Yahoo Finance [yfinance](https://pypi.org/project/yfinance/), [Panel](https://panel.holoviz.org/),[DuckDB](https://duckdb.org/), OpenAI's [Vision Model preview API](https://platform.openai.com/docs/guides/vision) and [ImageKit](https://docs.imagekit.io/getting-started/quickstart-guides/python/python_app). The app can be hosted on [Ploomber Cloud](https://www.platform.ploomber.io/).
+
+The app will store the plot generated and save it to ImageKit.io, and then use the OpenAI API to generate a response to the user's question about the plot. 
+
+### Pre-requisites
+
+1. OpenAI API key. Visit their [Documentation](https://platform.openai.com/docs/api-reference/introduction)
+2. ImageKit.io url endpoint, public key, and private key. Visit their [Dashboard](https://imagekit.io/dashboard)
```

Comment:
mention what the names of the environment variables should be

Reply:
Resolved 
https://github.com/ploomber/doc/pull/124/commits/5e842bf34e5934baf4a5733646b3edfdfd769d79

Reply:
resolved 
https://github.com/ploomber/doc/pull/124/commits/5134a06dd9966b18ae5cc687803e19a4ddf4f0c9